node/calico-windows.yaml

```diff
@@ -0,0 +1,104 @@
+apiVersion: apps/v1
```

Comment:
I've added this WIP manifest here so it's easier to see how this all hangs together. The final place for this will be in the calico docs dir as a template.